doc/source/_pycompwa/pycompwa.rst

```diff
@@ -1,19 +0,0 @@
-pycompwa package
```

Comment:
why did you delete these two files?

Reply:
Ah yeah, I thought at first they are just generated by `sphinx-apidoc` (which they are), but that would leave out the `ui` pybindings module.